packages/react/src/themes/teams/components/Icon/iconStyles.ts

```diff
@@ -150,10 +144,22 @@ const iconStyles: ComponentSlotStylesInput<IconProps, IconVariables> = {
         fill: v.disabledColor,
       }),
 
+      transform: `rotate(${rotate}deg)`,
+
       ...getSvgStyle('svg'),
     }
   },
 
+  flipInRtl: config => {
```

Comment:
as we've agreed on, lets rename it to `svgFlippedInRtl`, as now this set of styles has changed its semantics